DesignSafe-CI/portal
# Hotfix/pub pipeline typo

## Overview: ##
A user pointed out that after publishing, you're prompted with the following message, where 'area' is 'ared':

"Your project is being published. Do not attempt to publish your project again. Please wait until the project has finished publishing to edit or add anything in the project. You will find your published project in the "Published" ared of the Data Depot soon."

## PR Status: ##


## Related Jira tickets: ##

* [DES-XXX](https://jira.tacc.utexas.edu/browse/DES-XXX)

## Summary of Changes: ##
Changed 'ared' to 'area' 

## Testing Steps: ##
1. Go down the publishing pipeline. After you click publish, the message should appear without a typo. 

## UI Photos:
BEFORE:
![Screen Shot 2023-04-06 at 11 36 57 AM](https://user-images.githubusercontent.com/35277477/230442255-70ef6a03-d944-4bb4-8ec5-0a29699c1f4e.png)

AFTER:
![Screen Shot 2023-04-06 at 11 39 25 AM](https://user-images.githubusercontent.com/35277477/230442265-006b2eab-e215-47e9-970c-7d5d508e3755.png)
## Notes: ##
